starter-project-web-react/components/Button.jsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Button, { ButtonProps } from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import { purple } from '@mui/material/colors';
+
+const BootstrapButton = styled(Button)({
```

Comment:
do we need to use a bootstrap styled button?
it might be fine for now but if there is a material UI option that has less CSS, that might be preferable (more consistent)

we can leave it for now and change it if needed after review

Reply:
This is more a concern of "are we writing more code than needed" than what it would look like

Reply:
I did not use it. I thought I deleted it.